jest.preset.js

```diff
@@ -0,0 +1,28 @@
+// @ts-check
+
+const path = require('path');
+const { pathsToModuleNameMapper } = require('ts-jest/utils');
+
+const tsConfig = require('./tsconfig.base.json');
+
+const tsPathAliases = pathsToModuleNameMapper(tsConfig.compilerOptions.paths, {
+  prefix: `<rootDir>/${path.relative(process.cwd(), __dirname)}/`,
+});
+
+/**
+ * @type {jest.InitialOptions}
+ */
+const baseConfig = {
+  transform: {
+    '^.+\\.ts$': 'ts-jest',
```

Comment:
Why not include tsx in the transform by default, since it's so commonly used? Doesn't seem like it should hurt anything to have it listed here in packages that don't use tsx. (In general, if there are changes like this that we can make to reduce the number of manual overrides per package, I would very much prefer that in the interest of reducing maintenance headaches of having config duplicated tons of places.)

Reply:
I see the point, but how bad is that? To me it's not great, but I'd prefer the relatively low risk (and low consequence) of that occurring over the maintenance headaches that come from keeping a duplicated config in a whole bunch of places. (Not blocking on this though.)